packages/ui/src/server/controllers/login.ts

```diff
@@ -41,7 +40,20 @@ export async function handleLogin(req: Request, res: Response) {
                     res,
                     response,
                     undefined,
-                    (headers) => removeSecureFlagIfOriginInsecure(req, headers),
+                    (headers: Record<string, string[]>) => {
+                        if (headers['set-cookie']) {
+                            headers['set-cookie'] = headers['set-cookie'].concat(
+                                headers['set-cookie'].map((item) =>
+                                    item.replace(
+                                        YT_CYPRESS_COOKIE_NAME,
+                                        `${ytAuthCluster}:${YT_CYPRESS_COOKIE_NAME}`,
+                                    ),
+                                ),
+                            );
+                        }
+
+                        return removeSecureFlagIfOriginInsecure(req, headers);
```

Comment:
And now the function also should handle `${ytAuthCluster}:${YT_CYPRESS_COOKIE_NAME}` correctly.